adafruit_displayio_layout/widgets/flip_input.py

```diff
@@ -0,0 +1,625 @@
+        *,
+        value_list=None,
+        font=FONT,
+        color=0xFFFFFF,
+        value=0,  # initial value, index into the value_list
+        arrow_touch_padding=0,  # touch padding on the arrow sides of the Widget
+        arrow_color=None,
+        arrow_outline=None,
+        arrow_height=None,
+        arrow_width=None,
+        alt_touch_padding=0,  # touch padding on the non-arrow sides of the Widget
+        horizontal=True,
+        animation_time=None,
+        **kwargs,
+    ):
+
+        super().__init__(**kwargs, max_size=4)
+        # Group elements for the FlipInput.
+        # 0. The text
+        # 1. The group holding the temporary scroll bitmap
+        # 2. Up arrow: Triangle
+        # 3. Down arrow: Triangle
+
+        # initialize the Control superclass
+
+        # pylint: disable=bad-super-call
+        super(Control, self).__init__()
+
+        self.value_list = value_list
+        self._value = value
+
+        self._color = color
+        self._font = font
+        # preload the glyphs
+
+        self._arrow_touch_padding = arrow_touch_padding
+        self._alt_touch_padding = alt_touch_padding
+
+        self._horizontal = horizontal
+        self._display = display
+
+        self._animation_time = animation_time
+
+        # Find the maximum bounding box of the text and determine the
+        # baseline (x,y) start point (top, left)
+
+        left = None
+        right = None
+        top = None
+        bottom = None
+
+        for this_value in value_list:
+            xposition = 0
+
+            for i, character in enumerate(this_value):
+                glyph = self._font.get_glyph(ord(character))
+
+                if (
+                    i == 0
+                ):  # if it's the first character in the string, check the left value
+                    if left is None:
+                        left = glyph.dx
+                    else:
+                        left = min(left, glyph.dx)
+
+                if right is None:
+                    right = max(
+                        xposition + glyph.dx + glyph.width, xposition + glyph.shift_x
+                    )
+                else:
+                    right = max(
+                        right,
+                        xposition + glyph.dx + glyph.width,
+                        xposition + glyph.shift_x,
+                    )  # match bitmap_label
+
+                if top is None:
+                    top = -(glyph.height + glyph.dy)
+                else:
+                    top = min(top, -(glyph.height + glyph.dy))
+
+                if bottom is None:
+                    bottom = -glyph.dy
+                else:
+                    bottom = max(bottom, -glyph.dy)
+
+                xposition = xposition + glyph.shift_x
+
+        self._bounding_box = [0, 0, right - left, bottom - top]
+
+        # Create the text label
+        self._label = bitmap_label.Label(
+            text=value_list[value],
+            font=self._font,
```

Comment:
Maybe we could offer a way for the user to pass `font_scale` or similar and use it here so they have a way to increase the font size if they are using fonts other than this awesome 14 segment.

Reply:
Great suggestion.  I added an integer `font_scale` option.  Also, I updated the simplest example to use the default font with a `font_scale=5`for two of the widgets, and the special font for the third.  I can default the example all back to the default font if that is preferred.

Reply:
I like the idea of using some default font and some special font ones in the example.